### ISSUE TITLE
memoizee@0.4.16 is broken due to node requirement

### ISSUE THREAD

**Issue by j-fulbright** [medikoo/memoizee#135]
error memoizee@0.4.16: The engine "node" is incompatible with this module. Expected version ">=.0.12". Got "20.13.1"

**Comment by vladimir-salvador** [comment]
is this ok?
```
"node": ">=.0.12"
```

I tried with nodejs 21 and it blows up with the mentioned error

**Comment by galaxyreal** [comment]
You can add the following to your package.json file:

For yarn install:

```
"resolutions": {
  "memoizee": "0.4.15"
}
```
For npm install:

```
"overrides": {
  "memoizee": "0.4.15"
}
```

**Comment by simlu** [comment]
If you are facing this issue in cli with a global yarn install you use `--ignore-engines` like so

```yarn global add serverless@3.33.0 --ignore-engines```

**Comment by mayaashizumi-munn** [comment]
Thanks @simlu that worked 👍

**Comment by drazisil** [comment]
Do you happen to have a related fix commands for pnpm?

**Comment by j-fulbright** [comment]
> Do you happen to have a related fix commands for pnpm?

Check issue 136 for some other suggestions, if youre leveraging package json and not doing a direct global install

**Comment by hyunbinseo** [comment]
@drazisil For pnpm, use the `pnpm.overrides` field.

```jsonc
// package.json
{
  // ...
  "pnpm": {
    "overrides": {
      "memoizee": "0.4.15"
    }
  },
}
```

Reference https://pnpm.io/ko/package_json#pnpmoverrides

**Comment by thoroc** [comment]
PR in https://github.com/medikoo/memoizee/pull/137

**Comment by medikoo** [comment]
Thank you for raising, and sorry for the trouble (TIL: that you can break pipelines by misconfigured `engine` field).

Fixed and published with v0.4.17
